scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -145,6 +148,25 @@ def validate_environment_name(asset_config: assets.AssetConfig) -> int:
     return error_count
 
 
+def validate_dockerfile(environment_config: assets.EnvironmentConfig) -> int:
+    """Validate Dockerfile.
+
+    Args:
+        environment_config (EnvironmentConfig): Environment config.
+
+    Returns:
+        int: Number of errors.
+    """
+    error_count = 0
+    dockerfile = environment_config.get_dockerfile_contents()

```

Comment:
Should we handle non-UNIX EOL characters? Please add a test with Windows EOLs to be sure.